benchmark_runner/common/oc/oc.py

```
@@ -91,16 +91,27 @@ def get_virtctl_version(self):
-        version_tuple = tuple(map(int, version.split(".")))
+        cache_attr = '_virtctl_ge_cache'
+        if hasattr(self, cache_attr):
+            cached = getattr(self, cache_attr)
+            min_version_tuple = tuple(map(int, min_version.split(".")))
+            return cached >= min_version_tuple
+        try:
+            version = self.get_virtctl_version().lstrip("v").split("-")[0]
+            version_tuple = tuple(map(int, version.split(".")))
+            setattr(self, cache_attr, version_tuple)
+        except (ValueError, AttributeError):
+            logger.warning(f'Could not parse virtctl version, defaulting to pre-1.6.0 syntax')
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)